atoma-service/src/config.rs

```diff
@@ -53,6 +53,9 @@ pub struct AtomaServiceConfig {
 
     /// The environment for the Atoma Service.
     pub environment: Option<String>,
+
+    /// List of allowed sui addresses for fiat payments.
+    pub fiat_allowed: Vec<String>,
```

Comment:
can we make this a bit more descriptive? such as  `fiat_allow_listed_addresses` upon reading this I had thought this was to enable/disable fiat payments for everyone (not just select wallets)

Reply:
renamed to `fiat_allowed_addresses`